ExtraDry/ExtraDry.Blazor/Components/Dry/DryStaticFilter.razor

```diff
@@ -0,0 +1,5 @@
+﻿@namespace ExtraDry.Blazor
+@typeparam TItem
```

Comment:
Does the static filter absolutely require a type?  The QueryBuilder doesn't require one so this seems odd.  Perhaps if there was some strong typed lambda-style filters being set.  But if so, that might be a future upgrade.

Reply:
Removed as it is not currently being used.